pkg/playground/react-patterns.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="../base1/cockpit.css" type="text/css" rel="stylesheet">
     <script src="../base1/bundle.js"></script>
-    <script src="../base1/cockpit-components.js"></script>
+    <script src="../performance/bundle.js"></script>
```

Comment:
The end game here is to include the dialog in the base1 bundle right? If so, I think this is an ok interim measure.